components/style/macros.rs

```diff
@@ -128,11 +128,3 @@ macro_rules! local_name {
         $crate::values::AtomIdent(atom!($s))
     };
 }
-
-/// Asserts the size of a type at compile time.
-macro_rules! size_of_test {
-    ($t: ty, $expected_size: expr) => {
-        #[cfg(target_pointer_width = "64")]
-        const_assert_eq!(std::mem::size_of::<$t>(), $expected_size);
-    };
-}
```

Comment:
Let's keep this one around to avoid differing too much from upstream.

Reply:
OK, so I will keep this in the style crate, also add a copy into its own crate to be used by layout, msg, selectors and style_traits, and then move it upstream.